### ISSUE TITLE
Website breaks without any major changes

### ISSUE THREAD

**Issue by arefbt** [jbroadway/elefant#319]
My website broke twice and I had to reinstall the cms twice. I didn't make any major changes and even tried to replace the files but nothing helps. It's a bug which shows file uploader in frontend. http://www.monarta.com/ please check.

**Comment by lux** [comment]
It looks like you might have set the status of the index page to private, which could cause it to forward to the admin login page to password protect it.

**Comment by arefbt** [comment]
I have restored all the minimalist theme files to original but nothing happens however if I change the default theme to bigtype the website works again.

**Comment by lux** [comment]
Thanks for emailing me the additional details so I could take a closer look.

It seems that the [filemanager/photo](https://www.elefantcms.com/helper/filemanager/photo) helper was the cause of the issue, where somehow the templates were rendered as if that helper was missing and rendered the `filemanager/index` script instead.

I was able to fix it by editing the `minimal/inc/index` and `minimal/inc/navbar` templates, which cleared the cache and allowed the helper, which was embedded in both of those templates, to re-render properly. The templates themselves are unchanged, I just removed the tag, saved the templates, then added it back in and saved them again.

I believe that should fix the issue moving forward, but let me know if you run into any more issues.
